core/db/migrate/20210702112334_add_missing_timestamp_columns.rb

```diff
@@ -0,0 +1,37 @@
+class AddMissingTimestampColumns < ActiveRecord::Migration[5.2]
+  def change
+    # Add missing created_at columns
+    add_column :spree_promotion_actions, :created_at, :datetime
```

Comment:
@tomurb also let's use a condition here, eg.

```suggestion
    add_column :spree_promotion_actions, :created_at, :datetime
    add_column :spree_promotion_actions, :created_at, :datetime unless column_exists?(:spree_promotion_actions, :created_at)
```